CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.6 (TBD)
+* Enhancements
```

Comment:
This change also requires a corresponding change to the installation instruction Sphinx docs.  Additionally, the contributor’s guide needs to be updated to add this as a dependency.